docs/pages/tutorial/eas/introduction.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'EAS Tutorial: Introduction'
+sidebar_title: Introduction
+description: An introduction to the tutorial for building apps for Android and iOS using Expo Application Services (EAS) that covers the Build, Update, and Submit workflows.
+---
+
+import Highlight from '~/components/plugins/Highlight';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this tutorial, we'll take our app built in the [previous tutorial](/tutorial/introduction/), create a development build and a preview version, and deploy it to the app stores.
```

Comment:
```suggestion
In this tutorial, we'll take our app built in the [previous tutorial](/tutorial/introduction/), create a development build and a preview version, and deploy it to app stores.
```

(and also update other places where we say "the app stores" to just "app stores" - there are many app stores and google play / apple are just the biggest ones)